README.md

```diff
@@ -1,8 +1,9 @@
 # NodeObservability Operator [WIP]
 
-The `NodeObservability` Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent), these agents will be deployed as DaemonSets (on all or slected Nodes). The agent is used to trigger crio and kubelet profile data to the containers hostPath for later retrieval (this is the MVP). 
+The `NodeObservability` Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent), these agents will be deployed as DaemonSets (on all or slected Nodes). \
+The agent is used to trigger crio and kubelet profile data to the nodes hostPath for later retrieval (this is the MVP). 
 
-**Note**: This Operator is in the early stages of implementation.
+**Note**: This Operator is in the early stages of implementation, and will be changing.
```

Comment:
May I know what will be changing ?

Reply:
The operator needs to interact with the Machine Config Operator , which hasn't been added yet ( we are addressing this in the next sprint)